packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -418,6 +440,13 @@ class AttachDisk extends Component {
 
   _selectVdi = vdi => this.setState({ vdi })
 
+  _isVdiSrOnSameHost = createSelector(
+    () => this.props.checkSr,
+    () => this.props.srs,
```

Comment:
This is weird because you actually only need the VDI's SR's host but you're "watching" the whole `srs` collection.